.env.example

```
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:f19zo68w33Do134KQHaswe1MNHk+hq9lutXl0WScnT0=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_KEY key should go before the APP_NAME key

(UnorderedKey)